app/messages/send-notification.js

```diff
@@ -0,0 +1,18 @@
+import { v4 as uuidv4 } from 'uuid'
+import { NotifyClient } from 'notifications-node-client'
+
+const sendNotification = async (message) => {
+  const notifyClient = new NotifyClient(process.env.NOTIFY_API_KEY)
+
+  await notifyClient.sendEmail(
+    message.body.data.notifyTemplateId,
+    message.body.data.commsAddress, {
```

Comment:
This can be an array of email addresses.  So need to handle if `commsAddress` is one email as a string or an array object with multiple strings.

Reply:
As a suggestion, could put this in a `for ..of` loop.  Then can await.  Can wrap each call in a try catch that way too.

With what you have now you've got a slightly harder job of working out which of the calls fails and logging it accordingly.